RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -47,7 +46,7 @@
     static func configure(dependencyManager: DependencyManager) {
         self.dependencyManager = dependencyManager
 
-        inAppQueue.async(flags: .barrier) {
+        dependencyManager.resolve(type: DispatchQueue.self)?.async(flags: .barrier) {
```

Comment:
Can we prevent duplicating this verbose expression?
Maybe a computed property `inAppQueue` ?

Reply:
Done.